### PR TITLE
Add Many Strats to Pink Hopper Room

### PR DESCRIPTION
**Pull request by kjbranch**

### CODE REVIEW

File: region/brinstar/pink/Pink Brinstar Hopper Room.json [hunk 1]
```diff
@@ -364,7 +375,10 @@
         ]}
       ],
       "clearsObstacles": ["B"],
-      "note": "Perform a very precise maximum height Spring Ball jump from the doorframe onto the Grapple block."
+      "note": [
+        "Perform a very precise Spring Ball jump from the doorframe onto the Grapple block.",
+        "A spin jump or crouch jump can be used; an air ball helps."
```
Comment (osse101) [inline region/brinstar/pink/Pink Brinstar Hopper Room.json:380]:
```suggestion
        "A running spin jump or crouch jump can be used."
```

File: region/brinstar/pink/Pink Brinstar Hopper Room.json [hunk 2]
```diff
@@ -355,7 +362,11 @@
       "name": "Spring Ball from the Doorframe",
       "requires": [
         {"doorUnlockedAtNode": 1},
-        "h_canMaxHeightSpringBallJump",
+        "canTrickySpringBallJump",
+        {"or":[
+          "canTrickyJump",
+          "canLateralMidAirMorph"
```
Comment (osse101) [inline region/brinstar/pink/Pink Brinstar Hopper Room.json:368]: Its not really a lateralMidAirMorph.  A delayed morph, or an early morph and springfling movement boost could do it.  But lateralmidairmorph works as an alternative to a trickyJump 🤷

**Review by osse101 — COMMENTED** [review]
Another one is 2->1 `comeInRunning Speed + HiJump`

And almost every 2->1 strat looks to ignore that you take a hopper hit before you can make it to the door.

File: region/brinstar/pink/Pink Brinstar Hopper Room.json [hunk 3]
```diff
@@ -534,37 +794,58 @@
...
-          "explicitWeapons": ["PowerBomb", "ScrewAttack", "Wave"]
-        }}
-      ],
-      "clearsObstacles": ["A"],
-      "note": "With access to the right side of the room, it's possible to kill the enemies safely with Wave or a PB or screw attack once on the other side."
+        {"or": [
+          {"and": [
+            "canCarefulJump",
+            "canCameraManip"
+          ]},
```
Comment (osse101) [inline region/brinstar/pink/Pink Brinstar Hopper Room.json:811]: How does this work?

File: region/brinstar/pink/Pink Brinstar Hopper Room.json [hunk 4]
```diff
@@ -534,37 +794,58 @@
...
+          {"and": [
+            "ScrewAttack",
```
Comment (osse101) [inline region/brinstar/pink/Pink Brinstar Hopper Room.json:813]: With screw attack I would jump at the big hopper from above, and I usually don't get hit.
But it looks always safe to just jump down the left side while in screw.  Then jump again to kill everything.

File: region/brinstar/pink/Pink Brinstar Hopper Room.json (hunk 3 above)
Comment (kjbranch) [reply region/brinstar/pink/Pink Brinstar Hopper Room.json]: I thought I was setting them up well from the right, but looks like it wasnt a thing.

File: region/brinstar/pink/Pink Brinstar Hopper Room.json (hunk 4 above)
Comment (kjbranch) [reply region/brinstar/pink/Pink Brinstar Hopper Room.json]: Looks like you can pretty much just jump anywhere as long as you spin jump again very quickly. I mentioned the left side, so the small one will hit you on a fail.

File: region/brinstar/pink/Pink Brinstar Hopper Room.json [hunk 5]
```diff
@@ -537,6 +537,18 @@
       ],
       "clearsObstacles": ["A"]
     },
+    {
+      "link": [2, 2],
+      "name": "Plasma SBA Sidehopper Kill",
```
Comment (osse101) [inline region/brinstar/pink/Pink Brinstar Hopper Room.json:542]: 👏

**Pull request merged**